src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/comparison.cpp

```diff
@@ -50,8 +51,9 @@ void ComparisonLayerCPUTest::SetUp() {
         modelType = inferPrc;
     }
 
-    const auto primitiveType = getPrimitiveType(comparisonType);
-    const auto primiticePrcType = primitiveType == "ref" ? ov::element::f32.to_string() : modelType.to_string();
+    const auto primitiveType = getPrimitiveType(comparisonType, modelType);
+    const auto primiticePrcType =
+        ov::intel_cpu::any_of(primitiveType, "ref", "acl") ? ov::element::f32.to_string() : modelType.to_string();
```

Comment:
Could you please point out to what exact change makes riscv to actually check the primitive type?